fdbrpc/sim2.actor.cpp

```diff
@@ -223,6 +228,27 @@ struct SimClogging {
 		auto& u = clogPairUntil[std::make_pair(from, to)];
 		u = std::max(u, now() + t);
 	}
+
+	void unclogPair(const IPAddress& from, const IPAddress& to) {
+		auto pair = std::make_pair(from, to);
+		auto it = clogPairUntil.find(pair);
+		if (it != clogPairUntil.end()) {
```

Comment:
Nit: can just do `clogPairUntil.erase(pair)` before checking the existence?

Reply:
Done.